cloudigrade/util/aws.py

```diff
@@ -10,6 +10,37 @@
 logger = logging.getLogger(__name__)
 
 
+class InstanceState(enum.Enum):
+    """
+    Enumeration of EC2 Instance state codes.
+
+    See also:
+        https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_InstanceState.html
+
+    """
+
+    pending = 0
+    running = 16
+    shutting_down = 32
+    stopping = 64
+    stopped = 80
```

Comment:
We should just include all the values here instead of leaving one out.

```
    0 : pending
    16 : running
    32 : shutting-down
    48 : terminated
    64 : stopping
    80 : stopped
```

Reply:
Good catch. I'm not sure how I missed that.